pages/common/tree.md

```diff
@@ -14,14 +14,14 @@
 
 `tree -a`
 
-- Print human readable size of files:
+- Print the tree without indentation lines, showing the full path instead:
 
-`tree -h`
+`tree -i -f`
 
-- Print the full path for each file:
+- Print the size of each node next to it, in human-readable format, with folders displaying their cumulative size (as in the `du` command):
```

Comment:
I think we're good with `Print the full path for each file with human-readable size`. Anything other than that seems like over-explaining to me :'D